Specs/Renderer/TextureSpec.js

```diff
@@ -25,9 +25,9 @@ describe(
     var blueOverRedFlippedImage;
     var red16x16Image;
 
-    var greenDXTImage;
-    var greenPVRImage;
-    var greenETC1Image;
+    var greenKTX2Image;
+    var whiteBasisKTX2Image;
+    // var greenUASTCKTX2Image;
```

Comment:
There's a lot of commented code in this file. It will be easier to review once commented code is either deleted or uncommented. Which tests trigger the `simulateError` race condition?

Reply:
Yeah the commented out code is mainly remnants from the old KTX file. I'll remove them